client/src/components/BookCard.tsx

```diff
@@ -0,0 +1,35 @@
+
+import { Card, CardContent, Typography } from "@mui/material";
+import React from "react";
+import AuthorCard from "./AuthorCard";
+
+const BookCard = ({ bookData }: any) => {
+
```

Comment:
a nice pattern in JS is destructuring https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

So here we could do 

`const {bookName, pageCount, etc} = bookData`

You can them access them directly rather than via the objecet